### PR TITLE
feat: remove pullRequest/push event handlers

### PR DESCRIPTION
**Pull request by joachimroeleveld**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,17 +81,10 @@ on:
...
-      - labeled
-      - opened
-      - ready_for_review
-      - reopened
-      - synchronize
-      - unlabeled
-      - unlocked
-
+      - completed
 jobs:
```
Comment (JrSchild) [inline README.md:88]: You might want to add the if conclusion = successful check to demonstrate that it automatically merges if you don't add this check.

```
# Add any additional filters to indicate when the merge should be done.
if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

**Review by JrSchild — CHANGES_REQUESTED** [review]
Running this in the CI step doesn't work anymore so the first example under GitHub Actions should probably be changed.
And I'd leave the check_suite example as it was since it's still working fine.

File: README.md (hunk 1 above)
Comment (joachimroeleveld) [reply README.md:75]: 👍🏻 
Changed in 
bb0cc8e

File: README.md [hunk 2]
```diff
@@ -66,59 +56,24 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 ```
 
-### Third party CI systems
```
Comment (JrSchild) [inline README.md:69]: I would personally keep this old one here with the subtitle Third party CI systems. The check suite seems to have been intended for that purpose. So let's address workflow run first.

**Review by JrSchild — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (joachimroeleveld) [reply README.md:69]: I'm not sure, I think it was `workflow-run` that was intended for 3rd party CI systems, but now it has become default. I left it open for the consumer to decide which trigger to run, without suggesting the use case.

Comment (joachimroeleveld) [reply README.md:69]: But it makes more sense to have `workflow_run` by default I agree.

Comment (joachimroeleveld) [reply README.md:69]: But it makes more sense to have `workflow_run` by default I agree. 00b19b0

Comment (joachimroeleveld) [reply README.md:69]: @JrSchild I am going to merge this one as #844 depends on it. The second one I need to merge for all Dependabot PRs that will be rebased to succeed. If you have comments still I'll create another PR for it.

**Pull request merged**